# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/clang/no-clang-tidy.diff
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./clang-tools-extra/CMakeLists.txt.vanilla	2024-03-18 20:14:11.966218277 +0100
+++ ./clang-tools-extra/CMakeLists.txt	2024-03-18 20:14:44.732216626 +0100
@@ -16,7 +16,7 @@
 add_subdirectory(clang-apply-replacements)
 add_subdirectory(clang-reorder-fields)
 add_subdirectory(modularize)
-add_subdirectory(clang-tidy)
+#add_subdirectory(clang-tidy)
 
 add_subdirectory(clang-change-namespace)
 add_subdirectory(clang-doc)
@@ -38,7 +38,7 @@
 CMAKE_DEPENDENT_OPTION(CLANG_ENABLE_CLANGD "Build clangd language server" ON
                        "LLVM_ENABLE_THREADS" OFF)
 if (CLANG_ENABLE_CLANGD)
-  add_subdirectory(clangd)
+  #add_subdirectory(clangd)
 endif()
 
 # Add the common testsuite after all the tools.
